app/src/main/java/org/helfoome/presentation/restaurant/RestaurantReviewTabFragment.kt

```diff
@@ -83,13 +84,19 @@ class RestaurantReviewTabFragment : BindingFragment<FragmentReviewBinding>(R.lay
     }
 
     private fun moveToBlog(review: BlogReviewInfo) {
-        if (Patterns.WEB_URL.matcher(review.url).matches()) {
-            try {
-                startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(review.url)))
-                return
-            } catch (e: ActivityNotFoundException) {
-                e.printStackTrace()
+        if (!Patterns.WEB_URL.matcher(review.url).matches()) return
+        try {
+            Intent(requireContext(), WebViewActivity::class.java).apply {
+                putExtra(ARG_WEB_VIEW_LINK, review.url)
+            }.let {
+                startActivity(it)
             }
```

Comment:
```suggestion
       startActivity(
            Intent(requireContext(), WebViewActivity::class.java).apply {
                putExtra(ARG_WEB_VIEW_LINK, review.url)
            }
       )
```
이렇게만 표기해도 괜찮을 것 같습니다!

Reply:
감사합니다 :)